examples/pipelines/controlnet-interior-design/components/segment_images/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/ml6team/fondant.git@main
```

Comment:
should numpy be here? or is it installed with transformers?

Reply:
Should be fine when Transformers is installed